header.html

```diff
@@ -0,0 +1,103 @@
+<header class="o-header-services o-header-services--b2b" data-o-component="o-header">
+	<div class="o-header-services__top o-header-services__container">
+		<div class="o-header-services__hamburger">
+			<a class="o-header-services__hamburger-icon" href="#o-header-drawer" aria-controls="o-header-drawer">
+				<span class="o-header__visually-hidden">Menu</span>
+			</a>
+		</div>
+		<div class="o-header-services__ftlogo"></div>
+		<div class="o-header-services__title">
+			<h1 class="o-header-services__product-name">{{ nav.heading }}</h1>
+		</div>
+		{{#if @root.licenceList.length }}
+		<div class="o-header-services__related-content licence-switcher__container">
+			<span class="o-header-services__related-content-link">
+				<div class="o-forms licence-switcher__form">
+					<form action="/overview/licenceSwitch" method="POST" class="o-forms__affix-wrapper">
+						<select id="licenceSwitcher" name="licenceSelection" class="o-forms__select">
+							{{#each @root.licenceList}}
+							<option value="{{licenceId}}" {{#ifEquals licenceId @root.LICENCE_ID }}selected="true"{{/ifEquals}}>{{contractId}} - {{name}}</option>
+							{{/each}}
+						</select>
+						<div class="o-forms__suffix n-util-hide-js">
+							<button type="submit" class="licence-switcher__submit">Go To Licence</button>
+						</div>
+					</form>
+				</div>
+			</span>
+		</div>
+		{{/if}}
+	</div>
+	<nav class="o-header-services__primary-nav">
+		<div class="o-header-services__container">
+			<ul class="o-header-services__nav-list">
+				{{#each nav.items}}
+				{{#if flag}}
+					{{#if (lookup @root.flags flag)}}
+						<li class="o-header-services__nav-item {{#if last}}o-header-services__nav-item--last kmt-text--transform-none{{/if}}">
+							<a class="o-header-services__nav-link {{#if selected}}o-header-services__nav-link--selected{{/if}}" href="{{href}}">
```

Comment:
Don't we need to add `data-trackable` here?

Reply:
Good spot, yes we do!